### DISCUSSION TITLE
How To Retrieve the authenticated user info

### DISCUSSION THREAD

**Discussion by ghost** [keystonejs/keystone#8051] [Questions]
i have gone through the doc and the authenicatedITem doesn't work, instead it throws the below error 

```An error occured while running \"Access control\".\n  - User.access.filter.query: Variable \"$where\" got invalid value { user: { id: [Object] } }; Field \"user\" is not defined by type \"UserWhereInput\".",```

how do i resolve this

**Comment by ghost** [comment]
![Uploading Screenshot (684).png…]()

**Reply by flexdinesh** [reply]
The session data is attached to the context object and you can access it as `context.session`. [Take a look at this example](https://github.com/keystonejs/keystone/blob/5525800c8568a2d093fd6103bc5147363e66814d/examples/ecommerce/mutations/addToCart.ts#L11-L12).
